app/lib/services/voip/MediaCallEvents.ios.test.ts

```
@@ -1,16 +1,70 @@
 /**
  * @jest-environment node
  *
- * iOS-only mute tests: requires isIOS = true so didPerformSetMutedCallAction listener is registered.
+ * iOS-only paths: isIOS = true, NativeEventEmitter for VoIP events, CallKit listeners.
  */
-import { setupMediaCallEvents } from './MediaCallEvents';
+import type { VoipPayload } from '../../../definitions/Voip';
```

ESLint

[error] There should be at least one empty line between import groups

(import/order)